apps/docs/.vitepress/config.ts

```diff
@@ -214,10 +214,6 @@ export default defineConfig({
                 text: 'Signing',
                 link: '/guide/wallets/signing',
               },
-              {
-                text: 'Test Wallets',
-                link: '/guide/wallets/test-wallets',
-              },
```

Comment:
I think we should still have documentation around creating _just_ test wallets. There could be cases where someone wants to create a test wallet but has their own node running.

Reply:
That is the testing scenario that I'd like to avoid after introducing `launchTestNode`. I want our users to isolate their tests on the node level and, if they want to have multiple wallets, they can use the `{walletConfig: {...}}` option for it. If they don't use this utility but rather use their own node _for tests_, they'll come across the same problems our SDK came across around test isolation, one test spending UTXOs and another failing because it wanted to spend them as well, etc.